terraform_modules/step_function/step_function_definitions/tag_auditing.tpl

```diff
@@ -9,7 +9,14 @@
         %{ if tag_auditor_init != "" ~}
         "Init": {
             "Type": "Task",
-            "Resource": "${tag_auditor_init}",
+            "Resource": "arn:aws:states:::lambda:invoke",
+            "Parameters": {
+                "FunctionName": "${tag_auditor_init}",
```

Comment:
does this work with an arn? this may be bad variable naming but "tag_auditor_init" is the arn.

Reply:
yes. looks weird but per AWS's example https://docs.aws.amazon.com/step-functions/latest/dg/input-output-contextobject.html it is passed like that
also tested & working